website/docs/r/policy_predefined_gateway_policy.html.markdown

```diff
@@ -0,0 +1,101 @@
+* `tag` - (Optional) A list of scope + tag pairs to associate with this Gateway Policy.
+* `rule` (Optional) A repeatable block to specify rules for the Gateway Policy. This setting is applicable to non-Default policies only. This setting is not applicable to policy belonging to `DEFAULT` category. Each rule includes the following fields:
+  * `display_name` - (Required) Display name of the resource.
+  * `description` - (Optional) Description of the resource.
+  * `destination_groups` - (Optional) A list of destination group paths to use for the policy.
+  * `destinations_excluded` - (Optional) A boolean value indicating negation of destination groups.
+  * `direction` - (Optional) The traffic direction for the policy. Must be one of: `IN`, `OUT` or `IN_OUT`. Defaults to `IN_OUT`.
+  * `disabled` - (Optional) A boolean value to indicate the rule is disabled. Defaults to `false`.
+  * `ip_version` - (Optional) The IP Protocol for the rule. Must be one of: `IPV4`, `IPV6` or `IPV4_IPV6`. Defaults to `IPV4_IPV6`.
+  * `logged` - (Optional) A boolean flag to enable packet logging.
+  * `notes` - (Optional) Text for additional notes on changes for the rule.
+  * `profiles` - (Optional) A list of profiles for the rule.
+  * `scope` - (Required) List of policy paths where the rule is applied.
+  * `services` - (Optional) List of services to match.
+  * `source_groups` - (Optional) A list of source group paths to use for the policy.
+  * `source_excluded` - (Optional) A boolean value indicating negation of source groups.
+  * `log_label` - (Optional) Additional information (string) which will be propagated to the rule syslog.
+  * `tag` - (Optional) A list of scope + tag pairs to associate with this Rule.
+  * `action` - (Optional) The action for the Rule. Must be one of: `ALLOW`, `DROP` or `REJECT`. Defaults to `ALLOW`.
+* `default_rule` (Optional) A repeatable block to modify default rules for the Gateway Policy in a `DEFAULT` category. This setting is only applicable to policies in `DEFAULT` category. Each rule includes the following fields:
+  * `scope` - (Required) Scope for the default rule that should be modified. Only one default rule can be present for each scope.
+  * `description` - (Optional) Description of the resource.
+  * `logged` - (Optional) A boolean flag to enable packet logging.
+  * `log_label` - (Optional) Additional information (string) which will be propagated to the rule syslog.
+  * `tag` - (Optional) A list of scope + tag pairs to associate with this Rule.
+  * `action` - (Optional) The action for the Rule. Must be one of: `ALLOW`, `DROP` or `REJECT`.
+
+## Attributes Reference
+
+In addition to arguments listed above, the following attributes are exported:
+
+* `id` - ID of the Secuirty Policy.
+* `revision` - Indicates current revision number of the object as seen by NSX-T API server. This attribute can be useful for debugging.
+* `path` - The NSX path of the policy resource.
+* `rule`:
```

Comment:
I think those attributes refer to the default rules too?

Reply:
Done